Tests/DependencyInjection/Fixtures/config/yml/full.yml

```diff
@@ -42,6 +42,15 @@ doctrine_mongodb:
                 slaveOkay:        true
                 socketTimeoutMS:  1000
                 ssl:              true
+                tls:                                 true
+                tlsAllowInvalidCertificates:          false
+                tlsAllowInvalidHostnames:            false
+                tlsCAFile:                           '/path/to/cert.pem'
+                tlsCertificateKeyFile:                '/path/to/key.crt'
+                tlsCertificateKeyFilePassword:        'secret'
+                tlsDisableCertificateRevocationCheck: false
+                tlsDisableOCSPEndpointCheck:         false
+                tlsInsecure:                         false
```

Comment:
and those values here (at least one to another, a space off here and there really itches ;)

Reply:
Done